index.html

```diff
@@ -163,12 +239,28 @@
       .attr('r', 5);
 
     points
-      .attr('cx', function (d) { return x(d.unemployment); })
-      .attr('cy', function (d) { return y(d.inflation); });
+      .attr('cx', function (d) { return x(d.year); })
+      .attr('cy', function (d) { return y(d.value); });
 
     points.exit().remove();
 
```

Comment:
Have you defined binCount in this file anywhere? Note that the histogram layout is trying to reference that.